.eslintrc.js

```diff
@@ -392,6 +392,7 @@ module.exports = {
 			rules: {
 				'@wordpress/no-global-active-element': 'off',
 				'@wordpress/no-global-get-selection': 'off',
+				'no-page-pause': 'error',
```

Comment:
Let's try namespacing it.

```suggestion
				'playwright/no-page-pause': 'error',
```

Reply:
mmh, that still allows the pauses on the tests, I'm not getting any errors when I try to commit them

Reply:
Hmm, it's working for me though. 🤔